app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/ProductViewActivity.kt

```diff
@@ -262,7 +262,7 @@ class ProductViewActivity : BaseActivity(), OnRefreshListener {
                 adapter.addFragment(ContributorsFragment.newInstance(productState), activity.getString(R.string.contribution_tab))
             }
 
-            if (isFlavors(OFF)) adapter.addFragment(ServerAttributesFragment().applyBundle(fBundle), "Server attributes")
+            if (isFlavors(OFF)) adapter.addFragment(ServerAttributesFragment().applyBundle(fBundle), "Synthesis")
```

Comment:
I initially used a string but it would be better if we used a resource. Could you add this to the title names array in res?

Reply:
On it.

Reply:
I made the change.